src/hotspot/cpu/aarch64/templateInterpreterGenerator_aarch64.cpp

```diff
@@ -896,9 +900,11 @@ void TemplateInterpreterGenerator::generate_fixed_frame(bool native_call) {
   __ stp(rfp, lr, Address(sp, 10 * wordSize));
   __ lea(rfp, Address(sp, 10 * wordSize));
 
-  __ ldr(rcpool, Address(rmethod, Method::const_offset()));
-  __ ldr(rcpool, Address(rcpool, ConstMethod::constants_offset()));
-  __ ldr(rcpool, Address(rcpool, ConstantPool::cache_offset()));
+  // Save ConstantPool* in r11_constants for later use to avoid loading multiple times
+  Register r11_constants = r11;
+  const Register r11 = noreg;
```

Comment:
I understand the motivation here, but I really wouldn't want to see this practice deployed any further across the AArch64 back end. I have to read a _lot_ of code in `cpu/aarch64`, and it's confusing enough already. :-)
Rather that this, you could just use another register from the dozen or so that are free.